---
title: Visualization, Differential Expression Analysis and Downstream Analysis of
  Glioma Transcriptomics Count Data
author: "Nishat Tamanna"
date: "2024-09-10"
output:
  html_document: default
  pdf_document: default
---
The whole analysis will use a [count dataset of glioblastoma transcriptomic samples](https://raw.githubusercontent.com/HackBio-Internship/public_datasets/main/Cancer2024/glioblastoma.csv). This dataset contains 5 samples from Recurrent Tumor and 5 from Primary Tumor. 

# Installing gplots for visualization

```{r, echo=TRUE, message=FALSE, warning=FALSE, results='hide'}
install.packages("gplots")
library("gplots")
```

# Generating Matrix from a CSV file

```{r}
raw_counts <- read.csv('/home/hp/Documents/HackbioCancer/glioblastoma_count_file.csv', row.names=1)
mat <- as.matrix(raw_counts)
log_data_matrix <- log2(mat+1) #log transformation of matrix as data range is too broad
head(log_data_matrix)
```

# Heatmap using diverging color palette

```{r, fig.width=10, fig.height=10}
col_palette <- colorRampPalette(c('blue','yellow','red'))(n=300)
heatmap.2(x=log_data_matrix, col = col_palette, density.info = 'none')
```

# Heatmap using sequential color palette

```{r, fig.width=10, fig.height=10}
col_palette <- colorRampPalette(c('blue','yellow'))(n=300)
heatmap.2(x=log_data_matrix, col = col_palette, density.info = 'none')
```

# Clustering Genes (rows) Alone

```{r, fig.width=10, fig.height=10}
col_palette <- colorRampPalette(c('blue','yellow','red'))(n=300)
heatmap.2(x=log_data_matrix, col = col_palette, density.info = 'none', dendrogram = 'row', Rowv = TRUE, Colv = FALSE)
```

# Clustering Samples (columns) Alone

```{r,fig.width=10, fig.height=10}
par(mar = c(10, 10, 10, 10))
col_palette <- colorRampPalette(c('blue','yellow','red'))(n=300)
heatmap.2(x=log_data_matrix, col = col_palette, density.info = 'none', dendrogram = 'column', Rowv = FALSE, Colv = TRUE, lmat = rbind(c(4, 3), c(2, 1)),  # Layout matrix
          lhei = c(1, 8),                  # Increase the height of the heatmap body
          lwid = c(0.2, 1),                  # Adjust widths for the heatmap and key
          cexCol = 0.7, srtCol = 20)
```

# Clustering both Genes and Samples

```{r,fig.width=10, fig.height=10}
col_palette <- colorRampPalette(c('blue','yellow','red'))(n=300)
heatmap.2(x=log_data_matrix, col = col_palette, density.info = 'none', Rowv = TRUE, Colv = TRUE)
```

# Installing DeSeq2 package for differential expression analysis

```{r, echo=TRUE, message=FALSE, warning=FALSE, results='hide'}
if (!require("BiocManager", quietly = TRUE))
    install.packages("BiocManager")

BiocManager::install("DESeq2")
library("DESeq2")
```

# Preparation for Differential Expression Analysis

```{r}
#Creating Sample Table
sample_table <- data.frame(
  sampleName <- colnames(raw_counts),
  condition = c("Recurret Tumor","Recurret Tumor","Recurret Tumor","Recurret Tumor","Recurret Tumor","Primary Tumor","Primary Tumor","Primary Tumor","Primary Tumor","Primary Tumor")
)
sample_table$condition <- as.factor(sample_table$condition)
head(sample_table)
```

# Creating DESeq Dataset and Running Differential Expression Analysis

```{r, echo=TRUE, message=FALSE, warning=FALSE, results='hide'}
dds <- DESeqDataSetFromMatrix(countData = raw_counts, colData = sample_table, design = ~ condition)
dds <- DESeq(dds)
```

# Filtering out Significant Genes (padj <0.05 and fold change cutoff 2)

```{r}
results_table <- results(dds)
significant_genes <- subset(results_table, padj < 0.05)
significant_genes_up <- subset(results_table, padj < 0.05 & log2FoldChange > 1)
significant_genes_down <- subset(results_table, padj < 0.05 & log2FoldChange < -1)
```

# Exporting genes, fold change and adjacent p values to CSV file

```{r}
background_genes <- data.frame(gene = rownames(results_table), log2FC = results_table$log2FoldChange, pval = results_table$pvalue)
DE_significantly_up_genes <- data.frame(gene = rownames(significant_genes_up), log2FC = significant_genes_up$log2FoldChange, pval = significant_genes_up$pvalue)
DE_significantly_down_genes <- data.frame(gene = rownames(significant_genes_down), log2FC = significant_genes_down$log2FoldChange, pval = significant_genes_down$pvalue)
head(DE_significantly_up_genes)
# Exporting into a CSV file
write.csv(background_genes, "background_genes.csv", row.names = FALSE)
write.csv(DE_significantly_up_genes, "DE_significantly_up_genes.csv", row.names = FALSE)
write.csv(DE_significantly_down_genes, "DE_significantly_down_genes.csv", row.names = FALSE)
```

# Functional Enrichment Analysis with ShinyGO
Using the default parameters in [ShinyGO 0.80](http://bioinformatics.sdstate.edu/go/), for 89 significantly differential expressed genes, we found the following top 10 enriched KEGG pathways.

```{r, echo=FALSE, fig.width=10, fig.height=10}
knitr::include_graphics("/home/hp/Documents/HackbioCancer/enrichment_plot.png")
```


