**Summary of the publication: “Integrated genomic analyses of ovarian carcinoma”**

Authors: Nishat Tamanna, Menatalla Hisham Hassan Attia

Ovarian cancer is the fifth leading factor to death from cancer in American women. The available standard treatment is aggressive surgery which leads to only 31% as five-year survival probability and recurrence of cancer in 25% patients within 6 months. The lack of successful treatment strategies led the Cancer Genome Atlas (TCGA) researchers to analyze annotated High Grade Serous Ovarian Cancer (HGS-OvCa) samples to identify molecular abnormalities influencing pathophysiology, affecting outcome and constituting therapeutic targets.

**_Mutation analysis_**

Exome sequencing of 316 outlined 19,356 somatic mutations, \~61 per tumor. Nine genes had significantly increased frequency of non-synonymous or splice site mutation impact which were TP53, BRCA1, BRCA2, RB1, NF1, FAT3, CSMD3, GABRA6 and CDK12; TP53 being the most frequently mutated, in 96%.

A list of \~600 genes that were less commonly mutated but of significance were identified by comparing the results with other databases which included mutations in BRAF, PIK3CA, KRAS and NRAS. Further analysis predicted 122 missense mutations to be oncogenic and twenty-seven percent of them to affect protein function.

**_Copy number analysis_**

Somatic copy number alterations (SCNAs) analysis in 489 HGS-Ovca genomes found 5 out of 8 recurrent gains and 18 out of 22 losses of regional aberrations occurring in more than 50% of the tumours. CCNE1, MYC, and MECOM were mostly encoded genes in 63 regions of recurrent focal amplification. There are therapeutic targets such as CCNE1, MECOM, MAPK1, and KRAS which appear to be amplified in approximately 10% of cases. Additionally, 50 focal deletions were identified which included key tumor suppressor genes like **PTEN**, **RB1**, and **NF1** and cell cycle control gene **CREBBP**.

**_mRNA and miRNA expression and DNA methylation analysis_**

Clustering analysis of 11,864 genes resulting in \~1,500 variable genes, revealed four distinct subtypes: immunoreactive, differentiated, proliferative, and mesenchymal; named based on the gene content. A similar analysis on a separate dataset confirmed these subtypes in HGS-OvCa. The immunoreactive subtype involves T-cell-related genes, the proliferative subtype shows high proliferation markers, the differentiated subtype is linked to ovarian tumor and secretory markers, and the mesenchymal subtype is characterized by genes associated with stromal and myofibroblast activity.

A 193-gene transcriptional signature predictive of overall survival was defined using the integrated expression dataset from 215 samples with 108 genes for poor survival and 85 for good survival. Kaplan–Meier survival analysis confirmed this association with 4 more validation datasets. Consensus clustering of miRNA expression identified three subtypes. miRNA subtype 1 overlapped with the miRNA proliferative subtype, and miRNA subtype 2 overlapped the miRNA mesenchymal subtype. Survival duration differed significantly between miRNA subtypes: with subtype-1 patients’ longer survival.

Increased DNA methylation and reduced tumor expression implicated 168 genes as epigenetically silenced in HGS-OvCa samples compared with fallopian tube controls and included RAB25 gene and AMT, CCL21, SPARCL1, BRCA1 promoters. Consensus clustering of variable DNA methylation across tumors identified four subtypes that were significantly associated with differences in age, BRCA inactivation events and survival. Survival duration did not differ significantly for transcriptional subtypes in the TCGA dataset. The proliferative group showed a decrease in the rate of MYC amplification and RB1 deletion, whereas the immunoreactive subtype showed an increased frequency of MECOM amplification.

**_Pathways influencing disease_** 

Pathway analysis of Integrated data from 316 fully analyzed cases showed dysregulated RB1 and PI3K/RAS, NOTCH signaling pathways in 67%, 45%, 22% and FOXM1 overexpression in 87% of cases, respectively. This study suggested patient survival depends on the mechanism of BRCA1/2 inactivation (mutation/methylation). Besides BRCA1/2, other altered homologous recombination genes contributing sensitiveness to PARP inhibitors were also found. Overall, homologous recombination defects may be present in approximately 50% of HGS-OvCa cases, providing a rationale for clinical trials of PARP inhibitors targeting tumors with aberrations.

In summary, the commonly deregulated pathways found, provide opportunities for therapeutic treatment. Again, inhibitors already exist for 22 genes in regions of recurrent amplification warranting assessment in HGS-OvCa cases where the target genes are amplified. Overall, these discoveries set the stage for approaches to the treatment of HGS-OvCa in which aberrant genes or networks are detected and targeted with therapies selected to be effective against these specific aberrations.

**_Reference_**

Cancer Genome Atlas Research Network. Integrated genomic analyses of ovarian carcinoma. Nature. 2011 Jun 29;474(7353):609-15. doi: 10.1038/nature10166. Erratum in: Nature. 2012 Oct 11;490(7419):298. PMID: 21720365; PMCID: PMC3163504.
